cinema/urls.py

```diff
@@ -1,10 +1,27 @@
-from django.urls import path
+from django.urls import path, include
 
-from cinema.views import movie_list, movie_detail
+from rest_framework import routers
+
+from cinema.views import (MovieViewSet,
+                          GenreList,
+                          GenreDetail,
+                          ActorList,
+                          ActorDetail,
+                          CinemaHallViewSet
+                          )
```

Comment:
fix everywhere 
you can use `black`
```suggestion
from cinema.views import (
    MovieViewSet,
    GenreList,
    GenreDetail,
    ActorList,
    ActorDetail,
    CinemaHallViewSet
)
```